auth-server/src/services/auth.ts

```diff
@@ -160,6 +244,87 @@ export default class AuthService {
 			throw new ForbiddenError(messages.ADDRESS_LOGIN_NOT_FOUND);
 		}
 
+		await redisDel(getAddressLoginKey(address));
+
+		return {
+			refreshToken: await this.getRefreshToken(user),
+			token: await this.getSignedToken(user),
+			user: {
```

Comment:
This can be removed now, the front doesn't need this any more for login. All these info are in the JWT.

Reply:
This is a big refactor. Will do in seperate PR

Reply:
https://github.com/paritytech/polkassembly/issues/762